atlasdb-timelock-server/src/main/java/com/palantir/atlasdb/timelock/TimeLockServer.java

```diff
@@ -150,23 +150,24 @@ private static TimeLockServices createInvalidatingTimeLockServices(
                 TimeLockServices.class);
     }
 
-    private static Transport createTransport(Optional<SslConfiguration> optionalSecurity) {
+    private static Transport createTransport(Optional<AtomixSslConfiguration> optionalSecurity) {
         NettyTransport.Builder transport = NettyTransport.builder();
 
         if (!optionalSecurity.isPresent()) {
             return transport.build();
         }
 
-        SslConfiguration security = optionalSecurity.get();
+        AtomixSslConfiguration security = optionalSecurity.get();
         transport.withSsl()
-                .withTrustStorePath(security.trustStorePath().toString());
+                .withTrustStorePath(security.sslConfiguration().trustStorePath().toString())
```

Comment:
OK. I think it would be good to extract the `security.sslConfiguration()` out (maybe to `baseSslConfiguration` or something like that)? This is purely for readability purposes.